Source/NaturalLanguageClassifierV1/Models/Classification.swift

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension NaturalLanguageClassifierV1 {
+    
+    public struct Classification: JSONDecodable {
+        
+        /// Unique identifier for the classifier used.
+        public var classifierId: String
+        
+        /// Link to the classifier.
+        public var url: String
+        
+        /// The submitted phrase.
+        public var text: String
+        
+        /// The class with the highest confidence.
+        public var topClass: String
+        
+        /// An array classes of up to ten className-confidence pairs that are sorted in descending order of confidence. If there are fewer than 10 classes, the sum of the confidence values is 100%.
```

Comment:
This is a long line. Consider breaking it up and continuing the comment on a newline.

I've really struggled in Swift to deal with line length. It's a fairly verbose language, so it definitely requires more than the 80 columns used by other languages. I personally find it helpful to limit the line length, though, since I sometimes switch to other editors besides Xcode. (Xcode is nice in that automatically wraps the lines.)

SwiftLint starts to produce warnings for lines at a length of 100. So for our purposes, I think we want to limit the length to 100 columns. (Which means I need to fix a lot of my code, too!)